### PR TITLE
Release v2.39.0

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,33 @@
...
+  - Ensure that container commands do not trigger telemetry ([#9397](https://github.com/serverless/serverless/pull/9397)) ([85b9e53](https://github.com/serverless/serverless/commit/85b9e5319df48904664f966e988cc725116ce865)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Add `commandDurationMs` to payload ([#9401](https://github.com/serverless/serverless/pull/9401)) ([d647125](https://github.com/serverless/serverless/commit/d647125ff5daa07972675fd28690d42746ab223b)) ([Piotr Grzesik](https://github.com/pgrzesik))
+  - Add `commandOptionNames` to payload ([#9387](https://github.com/serverless/serverless/pull/9387)) ([f5b2b9b](https://github.com/serverless/serverless/commit/f5b2b9be395c9c2d3de4c4f91f991276bc22dc33)) ([Piotr Grzesik](https://github.com/pgrzesik))
+  - Do not report telemetry for help commands ([#9388](https://github.com/serverless/serverless/pull/9388)) ([d81730c](https://github.com/serverless/serverless/commit/d81730c219c70cd11d04a23ee217c4af3b320985)) ([Piotr Grzesik](https://github.com/pgrzesik))
```
Comment (medikoo) [inline CHANGELOG.md:25]: This was already the case in previous version. With this PR we've just fixed state temporarily in `master`, so I think it doesn't belong to CHANGELOG

**Review by medikoo — COMMENTED** [review]
Looks good, I've just proposed to remove one item, as it doesn't describe change against previous version

File: CHANGELOG.md (hunk 1 above)
Comment (pgrzesik) [reply CHANGELOG.md:25]: You're totally right - great catch :+1:

**Review by medikoo — APPROVED** [review]
👍

**Pull request merged**